lib/Elevate/OS.pm

```diff
@@ -0,0 +1,155 @@
+package Elevate::OS;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::OS
+
+Abstract interface to the OS to obviate the need for if-this-os-do-this-elsif-elsif-else tech debt
+
+=cut
+
+use cPstrict;
+
+use Log::Log4perl qw(:easy);
+
+use Elevate::OS::CentOS7;
+use Elevate::OS::CloudLinux7;
+
+use constant SUPPORTED_DISTROS => qw{
+  CentOS7
+  CloudLinux7
+};
+
+use constant PRETTY_SUPPORTED_DISTROS => (
+    'CentOS 7',
+    'CloudLinux 7',
+);
+
+use constant AVAILABLE_UPGRADE_PATHS_FOR_CENTOS_7 => qw{
```

Comment:
these constants belong in C7.pm and CL7.pm as a variable.

Reply:
this is done